source/core/NumericString.cpp

```diff
@@ -1291,6 +1293,39 @@ IntMax ScanIntValues(char formatChar, char* beginPointer, char** endPointer)
     return intValue;
 }
 
+std::string validTrueValues[5] = { "y", "t", "true", "on", "yes" };
+std::string validFalseValues[4] = { "f", "false", "off", "no" };
```

Comment:
CG/NCG also matches "n".  The documentation page you linked incorrectly omits this.

Reply:
ah, good find. Please file a documentation CAR